src/Core/API/editMovieAPI.js

```diff
@@ -0,0 +1,15 @@
+import api from "./axiosInstance";
+
+const editMovieAPI = async (payload) => {
+    const movie = payload.payload;
+    delete movie.url;
+
+    return api.put('/movies', {
+        ...movie
+    })
+    .then((res) => {
+        return res.data;
```

Comment:
Consider improving code consistency by either adding `console.log` to all "then"s or just using short arrow function form `.then((res) => res.data)`
Unless of course, it's not ESLint who's changing your code to this form :)